about/people.yml

```diff
@@ -457,4 +457,13 @@ data:
     brown_directory_uuid: 294d8182-b793-46ad-a6a8-6548b7378c2e
     bio: Sanjana is a data science intern working with the Computational Biology Core at CCV. She is concentrating in Computational Biology ('26) and is interested in genetics and public health. She also loves to read sci-fi, run, and spend time with friends!
     image: sanjana_main.jpg
+  - name: Marwan Ali
+    type: Part Time
+    team: Advanced Research Computing
+    subteam: Computational Biology Core
+    title: Data Science Intern
+    github_username: marwanali55
+    brown_directory_uuid: 80e72ea3-bce9-4685-bc2c-3041b55facfc
+    bio: Marwan is a summer intern working with the Computational Biology Core at CCV. He is an undergraduate student studying Applied Mathematics. He loves reading, playing piano, and writing stand-up comedy.
+    image: marwan_main.PNG
```

Comment:
This says the image will be a png file, but the images are jpgs. Take a look at the preview link (in a comment on the PR) and try to debug why your profile picture isn't showing up!

Reply:
Hi Ellen! Sorry about that; I just updated it to marwan_main.jpg. Please let me know if there are any other issues.